worker/orgfetch.js

```diff
@@ -1,6 +1,10 @@
 const db = require('../util/pghelper');
 const opvapi = require('../api/orgpackageversions');
 const orgsapi = require('../api/orgs');
+const request = require('request');
+const { logger } = require('../util/logger');
+
+const INTERNAL_ORGS = ['GS0', 'CS46', 'CS49'];
```

Comment:
These are instances, not orgs.  And we should make them a config variable with these three as default values.  e.g. `const INTERNAL_INSTANCES = JSON.parse(process.env.INTERNAL_INSTANCES || ['GS0', 'CS46', 'CS49']);
`